### ISSUE TITLE
Typing speed in chat seems slow on Windows?

### ISSUE THREAD

**Issue by clin1234** [rigormorrtiss/discordo#468]
When logging in, the typing speed is normal, but once logged in, typing any message is slow (1 character displayed in 2 seconds)

Has anyone else suffered from the issue?

**Comment by GitMurf** [comment]
Yes I am a new user to discordo and this is the first thing I noticed. Unfortunately it is unusable in this state. @clin1234 is this something new? have you used discordo in the past on Windows without the slow typing? curious if this is a new issue or just the way it is in Windows?

**Comment by clin1234** [comment]
> Yes I am a new user to discordo and this is the first thing I noticed. Unfortunately it is unusable in this state. @clin1234 is this something new? have you used discordo in the past on Windows without the slow typing? curious if this is a new issue or just the way it is in Windows?

I just installed it from source yesterday.

**Comment by RobinHirst11** [comment]
fine on my machine

**Comment by Brosbe** [comment]
installed from source just now and I'm experiencing the same slow typing speed
I tried a couple of things:
1. pwsh through wezterm
2. powershell
3. cmd
4. git bash

I noticed the cursor hopping around in git bash, so I tried making the window quite a bit which caused the typing speed to improve.
My guess is that the slow typing comes from the UI being redrawn on every keystroke

**Comment by Brosbe** [comment]
I tried installing discordo through wsl using the AUR as well and typing is smooth and responsive there
